test/administrative-sdkSpec.js

```diff
@@ -65,7 +65,7 @@ describe('Administrative SDK', () => {
     sdk.getPronunciationChallenge(1, 2);
     sdk.listPronunciationChallenges(1);
     sdk.deletePronunciationChallenge(1);
-    sdk.createSpeechChallenge(1);
+    sdk.createSpeechChallenge(1, 2);
```

Comment:
I get that, but it's more readable that way. Anyway, rest is fine, so feel free to merge.